File: FingoDriver/src/main/java/com/yelloco/fingodriver/models/networking/refund/RefundApi.java
package com.yelloco.fingodriver.models.networking.refund;

import java.util.Map;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.HeaderMap;
import retrofit2.http.Headers;
import retrofit2.http.POST;

public interface RefundApi
{
    @POST("refunds")
    Call<RefundResponse> refund(@HeaderMap Map<String, String> headers, @Body RefundRequest refundRequest);
}

File: FingoDriver/src/main/java/com/yelloco/fingodriver/models/networking/refund/RefundApi.kt
package com.yelloco.fingodriver.models.networking.refund

import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.HeaderMap
import retrofit2.http.POST

interface RefundApi
{
    @POST("refunds")
    fun refund(
        @HeaderMap headers: Map<String, String?>,
        @Body refundRequest: RefundRequest
    ): Call<RefundResponse?>?
}